packages/api/src/libs/action-client.ts

```diff
@@ -0,0 +1,40 @@
+import fetch from "isomorphic-fetch";
+import type { Json } from "..";
+import assert from "assert";
+import join from "url-join";
+
+// This is a basic client to allow you to call into the server in the form of
+// client(action,arg1,arg2,...etc) => Promise<Json>.
```

Comment:
We're building client functionality into the API package?

Should this go in a separate package? Seems like we'd ideally not require the API user to need to worry about what's going on behind the API to use it. For instance, if I were downloading this client package, it might be confusing for me to see a bunch of documentation around how to run the API.

Reply:
i didnt have a good place to put this file, its for francesco to reference when interfacing with the API.  it shouldn't belong here, or be exported in the api package, but i also didnt think it fit into the SDK.

Reply:
Maybe put it in a non-exported examples folder so it's clear it's not production code?

Reply:
thats a good idea, i will make a folder like examples, and TODO to move it out there once we have a FE library